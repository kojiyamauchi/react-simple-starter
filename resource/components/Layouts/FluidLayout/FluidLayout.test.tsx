/**
 * @jest-environment jsdom
 */
/*
  Fluid Layout Component Unit Test.
*/

import { FluidLayout } from './'
import { BrowserRouter } from 'react-router-dom'
import { render /* ,screen */ } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
// import userEvent from '@testing-library/user-event'

describe('Fluid Layout Component Unit Test', () => {
  it('Snap Shot Testing', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <FluidLayout pages="ui-test" />
      </BrowserRouter>
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
