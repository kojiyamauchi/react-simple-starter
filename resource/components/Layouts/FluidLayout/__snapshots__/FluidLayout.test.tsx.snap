// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fluid Layout Component Unit Test Snap Shot Testing 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL jqkawm ui-test layout fluid-layout"
  >
    <main
      class="main-content"
    />
  </div>
</DocumentFragment>
`;
